testcases/OpTestLparFreq.py

```diff
@@ -0,0 +1,155 @@
+            self.check_mode(self.powersave_check_list[i])
+
+    def ModeFreq_test(self):
+
+        # This test sets Powersave mode to different available modes and
+        # checks whether the frequency chages accordingly.
+
+        self.cv_HMC.run_command("chpwrmgmt -m %s -r sys -o enable -t %s" %
+                                (self.system_name, self.powersave_modes_list[0]))
+        time.sleep(5)
+        static_freq = self.get_freq()
+        self.cv_HMC.run_command("chpwrmgmt -m %s -r sys -o enable -t %s" %
+                                (self.system_name, self.powersave_modes_list[2]))
+        time.sleep(5)
+        max_perf_freq = self.get_freq()
+        self.cv_HMC.run_command("chpwrmgmt -m %s -r sys -o enable -t %s" %
+                                (self.system_name, self.powersave_modes_list[1]))
+        time.sleep(5)
+        dynamic_freq = self.get_freq()
+        print("Average Frequency with Static Powersave mode is %s" % static_freq)
+        print("Average Frequency with Dynamic Performance mode is %s" % dynamic_freq)
+        print("Average Frequency with Maximum Performance mode is %s" % max_perf_freq)
+        if dynamic_freq < static_freq:
+            print("Dynamic Performance mode frequency is less than Static Powersave mode frequency")
+            self.failed_test.append("Static Powersave mode frequency check failed")
+        if max_perf_freq < dynamic_freq:
+            print("Maximum Performance mode frequency is less than Dynamic Performance mode frequency")
+            self.failed_test.append("Maximum Performance mode frequency check failed")
+
+    def Lpar_shutdown_freq_test(self):
+
+        # This test Set Powersave mode to "Staic Power Saver" mode when lpar is deactivated
+        # and check if the frequency is set properly when the lpar boots up
+
+        self.cv_HMC.run_command("chsysstate -r lpar -m %s -o shutdown -n %s" %
+                                (self.system_name, self.lpar_name))
+        time.sleep(20)
+        self.cv_HMC.run_command("chpwrmgmt -m %s -r sys -o enable -t %s" %
+                                (self.system_name, self.powersave_modes_list[0]))
+        self.cv_HMC.run_command("chsysstate -r lpar -m %s -o on -b norm -n %s" %
+                                (self.system_name, self.lpar_name))
+        time.sleep(120)
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.check_mode(self.powersave_check_list[0])
+
+    def Lpar_Dynamic_freq_test(self):
+
+        # This test Set Powersave mode to "Dynamic Performance Mode" mode and
+        # check if frequency is set based on system utilisation.
+
+        self.cv_HMC.run_command("chpwrmgmt -m %s -r sys -o enable -t %s" % (self.system_name, self.powersave_modes_list[1]))
+        time.sleep(5)
+        freq_before = self.get_freq()
+        self.console.run_command("cd /root")
+        self.console.run_command("wget http://liquidtelecom.dl.sourceforge.net/project/ebizzy/ebizzy/0.3/ebizzy-0.3.tar.gz")
```

Comment:
can we run this workload command from a config file as that can helps two ways 
1- workload can be change like we can have other workload as well 
2- if ebizzy url is failed to fetch test case should not breaks and easy to adopt